app/controllers/roots_controller.rb

```diff
@@ -0,0 +1,7 @@
+class RootsController < ApplicationController
+  def index
+    @greeting = Root.all.sample
+
+    render json: { greeting: @greeting, status: 200 }
```

Comment:
* Kindly fix this issue I have as the front end doesn't render greetings but the backend API does 
![image](https://user-images.githubusercontent.com/52497246/211115138-cf1d8809-73ae-449e-bea5-34d4fe3e8b86.png)
![image](https://user-images.githubusercontent.com/52497246/211115153-e7089d31-6644-4c00-b88a-348fb29a616d.png)